Package.swift

```diff
@@ -125,7 +125,9 @@ let package = Package(
     .package(
       name: "GoogleUtilities",
       url: "https://github.com/google/GoogleUtilities.git",
-      "7.2.1" ..< "8.0.0"
+      // TODO: Move from `revision` to version below before publishing.
+      .revision("a59fd0181d8f7c2b4c815cd6cddf4c38a4ba35ee")
+//      "7.4.0" ..< "8.0.0"
```

Comment:
This tag won't be available until we publish GoogleUtilities.

Reply:
cc @paulb777 - how did we handle this in the past?

Reply:
We either need to publish GoogleUtilities sooner or update this later in the release cycle.

There are some options to explore from https://forums.swift.org/t/best-practices-for-integration-testing-and-releasing-multiple-packages/47006/11, but nothing too clear or easy.

Reply:
Happy to hold off temporarily and cherry-pick this when it's published. We should also probably set up a presubmit that prevents `.revision` from being checked in to prevent failures in the future